requirements.txt

```diff
@@ -0,0 +1,7 @@
+# ocrd includes opencv, numpy, shapely, click
+ocrd >= 2.20.1
+seaborn >= 0.11.0
```

Comment:
seaborn is a plotting library, may be worth checking why this is needed

Reply:
For plotting 😆 Mostly for debugging, can be controlled with CLI flags.

Reply:
:+1: